usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/MealData.java

```diff
@@ -0,0 +1,80 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MealData {
+    private String mealName;
+    private Double kcal;
+
+    public MealData(String mealName, double kcal) {
+        this.mealName = mealName;
+        this.kcal = kcal;
+    }
+
+    /**
+     * Returns the meal name
+     * @return Meal name
+     */
+    public String getMealName() {
+        return mealName;
+    }
+
+    /**
+     * Sets a new meal name
+     * @param mealName New meal name
+     */
+    public void setMealName(String mealName) {
+        this.mealName = mealName;
+    }
+
+    /**
+     * Returns the amount of kcalories
+     * @return Amount of kcalories
+     */
+    public Double getKcal() {
+        return kcal;
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5314325)